google-cloud-clients/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/BigtableDataClientTest.java

```diff
@@ -168,13 +225,26 @@ public void proxyMutateRowCallableTest() {
   @Test
   public void proxyMutateRowTest() {
     RowMutation request =
-        RowMutation.create("fake-table", "some-key")
-            .setCell("some-family", "fake-qualifier", "fake-value");
+            RowMutation.create("fake-table", "some-key")
+                    .setCell("some-family", "fake-qualifier", "fake-value");
 
     bigtableDataClient.mutateRowAsync(request);
     Mockito.verify(mockMutateRowCallable).futureCall(request);
   }
 
+  @Test
+  public void mutateRowTest() {
+    Mockito.when(mockMutateRowCallable.futureCall(any(RowMutation.class)))
+            .thenAnswer((Answer) (invocationOnMock) -> ApiFutures.immediateFuture(Empty.getDefaultInstance()));
```

Comment:
Unfortunately, we can't use lambda's yet as we are keeping Java 7 source compatibility.